command/repo.go

```diff
@@ -143,7 +143,7 @@ func addUpstreamRemote(parentRepo ghrepo.Interface, cloneURL string) error {
 	upstreamURL := fmt.Sprintf("https://github.com/%s.git", ghrepo.FullName(parentRepo))
 	cloneDir := path.Base(strings.TrimSuffix(cloneURL, ".git"))
 
-	cloneCmd := git.GitCommand("-C", cloneDir, "remote", "add", "upstream", upstreamURL)
+	cloneCmd := git.GitCommand("-C", cloneDir, "remote", "add", "-f", "upstream", upstreamURL)
```

Comment:
I was wondering if it might be possible – and preferred – to instead repurpose and use an adjusted version of `AddRemote`: https://github.com/cli/cli/blob/master/git/remote.go#L74-L76

Reply:
That is generally a good idea, but in this instance it would require us to change the signature of AddRemote to accept an additional (optional) directory parameter.

Since making that change isn't a marked improvement over your current approach, in my book, I would just say— this looks good, and leave it as is! Thank you so much for taking a stab at a fix 🎉